### PR TITLE
fix the regression of unit test

### LINKED ISSUE #88: TEST_P(VAAPICreateBuffer, CreateBufferWithOutData) failed
test_va_api_createbuffer.cpp line 171 doCreateContext(Resolution(1, 1)); not check the min resolution, it will failed. then cause CreateBuffer test failed.

### PR DESCRIPTION
**Pull request by QuPengfei**
this add the minium resolution check in the unit test

Fixes #88

Signed-off-by: Pengfei Qu <Pengfei.Qu@intel.com>

### CODE REVIEW

File: test/test_va_api_createbuffer.cpp [hunk 1]
```diff
@@ -168,7 +170,8 @@ TEST_P(VAAPICreateBuffer, CreateBufferWithOutData)
             // vaCreateContext input requires resolution, since this test
             // doesn't create surfaces, passing 1x1 resolution should provide
             // the desired result.
```
Comment (uartie) [inline test/test_va_api_createbuffer.cpp]: Please update the comment

File: test/test_va_api_createbuffer.cpp [hunk 2]
```diff
@@ -168,7 +170,8 @@ TEST_P(VAAPICreateBuffer, CreateBufferWithOutData)
...
-            doCreateContext(Resolution(1, 1));
+            doGetMinSurfaceResolution(currentTestInput.inputConfig.first, currentTestInput.inputConfig.second, minResolution);
```
Comment (uartie) [inline test/test_va_api_createbuffer.cpp]: Please wrap this line.

File: test/test_va_api_createbuffer.cpp (hunk 1 above)
Comment (QuPengfei) [reply test/test_va_api_createbuffer.cpp]: thanks

File: test/test_va_api_createbuffer.cpp (hunk 2 above)
Comment (QuPengfei) [reply test/test_va_api_createbuffer.cpp]: @uartie ,i am not sure what is the idea of wrap? would you explain it?

**Comment by QuPengfei** [comment]
update

File: test/test_va_api_createbuffer.cpp [hunk 3]
```diff
@@ -166,9 +168,13 @@ TEST_P(VAAPICreateBuffer, CreateBufferWithOutData)
                 currentTestInput.inputConfig.second);
 
             // vaCreateContext input requires resolution, since this test
-            // doesn't create surfaces, passing 1x1 resolution should provide
+            // doesn't create surfaces, passing min resolution should provide
             // the desired result.
-            doCreateContext(Resolution(1, 1));
+            doGetMinSurfaceResolution(
```
Comment (uartie) [inline test/test_va_api_createbuffer.cpp]: `doGetMinSurfaceResolution` was removed in recent commit.  Please rebase and use new` getMinMaxSurfaceResolution` instead.

**Pull request merged**